app/classifier.py

```diff
@@ -51,59 +49,66 @@ def _load_instance(self, path_to_model: str):
 
     def _load_vocab(self, path_to_vocab: str):
         with open(path_to_vocab, 'r') as f:
-            self.vocab = f.readlines()
+            self.vocab = [line.rstrip() for line in f.readlines() if line.rstrip() != '']
 
-    def classify(self, seq: str, model_name: str) -> [(str, float)]:
-        """ classify calculates and returns a particular sequence's topic probability vector.
+    def classify(self, seqs: List[str], model_name: str, fixed_threshold: Optional[float] = None) \
```

Comment:
Let's add a multi_classify() that calls classify() and keeps that method mostly as it was before.

Reply:
same as in embedder: setting up the data for multi-operation in a single TensforFlow call is non-trivial, so keeping the singular would lead to code duplication.

Reply:
Makes sense.